packages/oid4vci/src/type.ts

```
@@ -0,0 +1,257 @@
+
+  deferred_credential_handler?: (
+    dto: DeferredCredentialRequestDto,
+  ) => OrPromise<DeferredCredentialResponseDto>;
+
+  notification_handler?: (dto: NotificationRequestDto) => OrPromise<void>;
+
+  credential_offer_handler?: (
+    offerId: string,
+  ) => OrPromise<CredentialOfferByValue>;
+
+  token_handler?: (dto: TokenDto) => OrPromise<any>;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)